shared/constants/metametrics.ts

```diff
@@ -169,7 +169,7 @@ export type MetaMetricsEventFragment = {
   /**
    * The event category to use for both the success and failure events.
    */
-  category?: string;
+  category: string;
```

Comment:
These were set to optional since it was causing TS errors before. I was able to revert these here by fixing the expected updateEventFragment types